architecture/additional_concepts/storage.adoc

```diff
@@ -219,13 +221,14 @@ The access modes are:
 
 [IMPORTANT]
 ====
-It is important to note that a volume's AccessModes are descriptors of the volume's capabilities, they are not enforced constraints.
-The storage provider is responsible for runtime errors resulting from invalid use of the resource.
-
-For example, a GCE Persistent Disk has AccessModes *ReadWriteOnce* and *ReadOnlyMany*.  It is left to the user to mark their
-claims as read only if they wish to take advantage of the volume's ability for ROX.  Errors in the provider show up at runtime
-as mount errors.
-
+It is important to note that a volume's `*AccessModes*` are descriptors of the
+volume's capabilities, they are not enforced constraints. The storage provider
```

Comment:
Comma splice above... Is there a word missing from the sentence? Maybe:

"A volume's Accesmodes are descriptors of the volume's capabilities, but they are not enforced constraints."?